app/controllers/katello/api/v2/repositories_controller.rb

```diff
@@ -287,6 +287,7 @@ def sync_complete
     def remove_content
       fail _("No content ids provided") if @content.blank?
       respond_for_async :resource => sync_task(::Actions::Katello::Repository::RemoveContent, @repository, @content)
+      async_task(::Actions::Katello::Repository::CapsuleGenerateAndSync, @repository)
```

Comment:
To clarify what we mean, it should likely be added to: app/lib/actions/katello/repository/finish_upload.rb similar to the GenerateMetadata task